arkouda/message.py

```diff
@@ -131,6 +132,17 @@ def _build_list_param(key: str, val: list) -> ParameterObject:
             v = val if t == str.__name__ else [str(x) for x in val]
             return ParameterObject(key, ObjectType.LIST, t, json.dumps(v))
 
+    @staticmethod
+    @typechecked
+    def _build_dict_param(key: str, val: Dict) -> ParameterObject:
+        j = []
+        for k, v in val.items():
+            if not isinstance(k, str):
+                raise TypeError(f"Argument keys are required to be str. Found {type(key)}")
```

Comment:
i think the error message was missed here in the update, i believe it should be `type(k)`

Reply:
My bad. Fixed.